types.go

```diff
@@ -1,11 +1,14 @@
 package discordwebhook
 
+import "time"
+
 type Message struct {
 	Username        *string          `json:"username,omitempty"`
 	AvatarUrl       *string          `json:"avatar_url,omitempty"`
 	Content         *string          `json:"content,omitempty"`
 	Embeds          *[]Embed         `json:"embeds,omitempty"`
 	AllowedMentions *AllowedMentions `json:"allowed_mentions,omitempty"`
+	Timestamp       *time.Time       `json:"timestamp,omitempty"`
```

Comment:
What is the exact purpose of these new field?

Reply:
Oops my bad I left it there, it should be moved in type mbed struct {}, see #12